.github/workflows/ci.yml

```diff
@@ -150,6 +150,30 @@ jobs:
       - name: Run Clippy
         run: cargo lint -- -D warnings
 
+
+  ecosystemTest:
+    needs: lint
+    name: Lint with vscode repository
+    runs-on: ubuntu-latest
+    
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Rust
+        uses: ./.github/actions/rustup
+        with:
+          clippy: true
+      - name: Clone vscode
+        uses: actions/checkout@master
+        with:
+          repository: microsoft/vscode
+          path: ./vscode
+      - name: Set directory permission
+        run: chmod +rwx ./vscode
+      - name: Build oxlint
+        run: cargo build -p oxc_cli --bin oxlint
+      - name: Run oxlint on vscode
+        run: ./target/debug/oxlint --quiet --ignore-path ./vscode/.eslintignore ./vscode/ 
```

Comment:
Does it fail CI on panic?

Reply:
I have just pushed some changes. My proposed solution is to throw all the output from oxlint to a temporary file and use grep to check for a panic keyword. As far as I am concern, there isn't a good way to catch panic from rust. Please let me know if this solution is too hacky.